src/views/Orders.vue

```diff
@@ -27,24 +27,25 @@
               </ion-label>
               <div class="metadata">
                 <ion-badge v-if="order.orderDate" color="dark">{{ moment.utc(order.orderDate).fromNow() }}</ion-badge>
-                <ion-badge v-if="order.statusId !== 'ORDER_APPROVED'" color="danger">{{ $t('pending approval') }}</ion-badge>
+                <ion-badge v-if="order.orderStatusId !== 'ORDER_APPROVED'" color="danger">{{ $t('pending approval') }}</ion-badge>
               </div>
               <!-- TODO: Display the packed date of the orders, currently not getting the packed date from API-->
             </ion-item>
 
-            <ProductListItem v-for="item in getShipGroupItems(shipGroup, order.items)" :key="item.itemId" :item="item" />
+            <ProductListItem v-for="item in getShipGroupItems(shipGroup, order.items)" :key="item.productId" :item="item" />
 
+            <!-- TODO: handle for phone number -->
             <ion-item v-if="order.phoneNumber">
               <ion-icon :icon="callOutline" slot="start" />
               <ion-label>{{ order.phoneNumber }}</ion-label>
-              <ion-button fill="outline" slot="end" color="medium" @click="copyToClipboard(order.phoneNumber)">
+              <ion-button fill="outline" slot="end" color="medium" @click.stop="copyToClipboard(order.phoneNumber)">
```

Comment:
Why stopPropagation() is needed here?

Reply:
Because we are using @click event on the whole card which creates event Bubbling condition and after copying email and phone number in clipboard we are opening details page as well.